paper/paper.md

```diff
@@ -0,0 +1,333 @@
+Next, we hosted a workshop for instructors to pilot the curriculum.
+We collected feedback from the learners and instructors at the end of the pilot
+workshop, then held a smaller half-day sprint to revise the curriculum based on
+the feedback.
+Community members continue to create issues, make edits, and review pull
+requests for ongoing refinement of the curriculum.
+We have plans to teach more workshops with instructors who did not develop the
+curriculum to gather even more feedback.
+
+# Curriculum
+
+Our curriculum is tailored to people with no prior coding experience who want to
+learn how to use programming to perform data analysis (Figure 2).
+We aim to teach our learners not only the basics of performing data analysis,
+but also to provide them with a framework to perform rigorous reproducible
+research that can easily be shared with others.
+
+![Curriculum overview. Created with BioRender.com.](curriculum-overview.png)
+
+## Learning Objectives
+
+The key learning objectives for our curriculum are:
+
+1. Create clear and informative data visualizations in R, starting with messy data.
+1. Perform version control using the Unix shell and git.
+1. Create reproducible reports using R markdown.
+1. Share code with others on GitHub.
+
+We believe these skills provide a solid foundation from which learners can teach
+themselves any additional coding skills that they will need in the future.
+
+## Course Content
+
+Our curriculum consists of modules that cover setup, data analysis and
+visualization, version control, sharing code, and writing reports.
+We also provide a goal for the workshop: to create a fictitious report for the
+United Nations on the relationship between gross domestic product (GDP), life
+expectancy, and CO~2~ emissions.
+The R programming lessons take a "tidyverse first" approach
+[@robinson_teach_2017] so learners can meet this goal.
+The modules are:
+
+0. Setup
+1. Welcome
+1. R for plotting (uses the tidyverse R packages)
+1. The Unix shell
+1. Git and GitHub
+1. R for data analysis (uses the tidyverse R packages)
+1. Writing reports in R markdown (uses the rmarkdown R package [@xie_r_2018])
+1. Group practice exercises
+1. Where to go from here
+
+Each lesson builds on the previous lessons.
+The Unix shell, git, and GitHub are introduced using the files generated in the
+R for plotting lesson.
+The lesson content for subsequent modules is then intermittently committed and
+pushed to GitHub.
+The written report combines all of the skills learned previously to write a
+report that one could share with the United Nations.
+Then, the learners have a chance to put everything they have learned into
+practice by working in small groups on practice problems that cover the course
+content.
+Finally, we end with a short module recapping what the workshop covered and how
+learners can continue to get help and learn more after the workshop ends.
+
+## Instructional Design
```

Comment:
I understand why this section is here. It is a bit repetitive in terms of what was already said above